pkg/controller/plan/kubevirt.go

```diff
@@ -1105,17 +979,13 @@ func (r *KubeVirt) virtualMachine(vm *plan.VMStatus) (object *cnv.VirtualMachine
 			"originalName", originalName, "newName", vm.Name)
 	}
 
-	if r.Plan.IsSourceProviderOCP() {
+	var ok bool
+	object, ok = r.vmTemplate(vm)
```

Comment:
this introduces a change in behavior - we intentionally clear the 'template' related properties